nwbwidgets/utils/timeseries.py

```diff
@@ -56,9 +56,9 @@ def get_timeseries_maxt(node: TimeSeries) -> float:
     if node.timestamps is not None:
         return node.timestamps[-1]
     elif np.isnan(node.starting_time):
-        return len(node.data) / node.rate
+        return (len(node.data)-1) / node.rate
```

Comment:
for a timesries: with rate=1 and starting_time=0, len(data) = 10, 
the maxt should be 9 no?

Reply:
yeah, good catch